### PR TITLE
214 refactor diary

### PR DESCRIPTION
**Pull request by digitaldiina**
Changes at a glance:

- co-located conceptual info
- improved navigation and index pages
- updated category page with 4 new cards
- moved 3 tutorials from api/tutorials to diary/tutorials and changed to yml format

### CODE REVIEW

File: docs/diary/tutorials/rss-feed-vb/index.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+- title: Configure
+  durationInMinutes: 3
+  content: |
+    [!include[Content: Configuration](includes/steps/config.md)]
+- title: Summary
+  durationInMinutes: 1
+  content: |
+    [!include[Content: RSS results](includes/steps/rss-results.md)]
+
+    <a href="../../assets/downloads/api/rssfeedfinalvb.zip" download>Click to download source code (zip)</a>
```
Comment (SuperOfficeDevNet) [inline docs/diary/tutorials/rss-feed-vb/index.yml:97]: Download link fails... change to (it works):
```markdown
[Click to download source code (zip)](~/assets/downloads/api/rssfeedfinalvb.zip)
```

**Review by SuperOfficeDevNet — CHANGES_REQUESTED** [review]
Just have a few broken links to fix, then we can merge this in. 

Good work! 🥇

File: docs/diary/index.yml [hunk 2]
```diff
@@ -85,57 +99,35 @@ conceptualContent:
...
         url: ../automation/index.yml
-      - text: Triggers
-        itemType: concept
-        url: ../automation/trigger/index.md
+      - text: Trigger SalesAfterSaveAppointment
+        itemType: reference
+        url: ../automation/trigger/reference/CRMScript.Event.Trigger.yml#CRMScript_Event_Trigger_SalesAfterSaveAppointment
+      - text: Trigger SalesBeforeSaveAppointment
+        itemType: reference
+        url: ../automation/trigger/reference/CRMScript.Event.Trigger.yml#CRMScript_Event_Trigger_SalesBeforeSaveAppointmen
```
Comment (SuperOfficeDevNet) [inline docs/diary/index.yml:113]: This link needs fixing:
url: ../automation/trigger/reference/CRMScript.Event.Trigger.yml#CRMScript_Event_Trigger_SalesBeforeSaveAppointmen
to
url: ../automation/trigger/reference/CRMScript.Event.Trigger.yml#CRMScript_Event_Trigger_SalesBeforeSaveAppointment

**Comment by digitaldiina** [comment]
Links updated @SuperOfficeDevNet

**Pull request merged**